src/ShoppingFeedHelper.php

```diff
@@ -698,7 +698,11 @@ public static function clean_process_running( $group ) {
 	 * @return WC_Logger
 	 */
 	public static function get_logger() {
-		return new WC_Logger();
+		static $logger;
+		if ( null === $logger ) {
+			$logger = new WC_Logger();
```

Comment:
Plutôt wc_get_logger(); à mon avis, comme ça pas besoin de le stocker (cf https://developer.woocommerce.com/2017/01/26/improved-logging-in-woocommerce-2-7/)